### ISSUE TITLE
demo 3: updates

### ISSUE THREAD

**Issue by AyceKirboi** [burnedpopcorn/Sugary-Spire-Web-Port#2]
could a data.win suffice, or no?
get v3.2hotfix1 on here: https://archive.org/download/pizza-tower-demo3-engine

**Comment by burnedpopcorn** [comment]
No, because
1. This port uses a different file type, game.unx, which does basically the same thing as data.win, but they are still technically different
2. data.win is a file specially compiled for Windows, so just replacing game.unx with it will NEVER work.
3. There's also some other files that are included that would normally not exist on a Windows build that is needed for a Web build, as they would either be already integrated inside of data.win, or are just for allowing the game to even run

So, TLDR, the updated data.win is incompatible  because of a lot of reasons, and the only way to update it is to obtain the source code

Whether that be by downloading leaked source code, or by making a GMS2 Decompiler. I know one exists, but I have no access to it, and it might be Game Specific
